src/mame/taito/qix.cpp

```diff
@@ -1337,7 +1370,7 @@ static const uint8_t xor2_table[] =
 	99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,99,11, 6,99,
 };
 
-int qix_state::kram3_permut1(int idx, int value)
+static inline int permut1(int idx, int value)
```

Comment:
Same for these permutation functions – they’re only relevant for one system, and it’s better to reflect that in the name.